studio.assetmanager.ams.yaml

```diff
@@ -0,0 +1,77 @@
+app-id: studio.assetmanager.ams
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: ams
+finish-args:
+  # X11 performance
+  - --share=ipc
+  # Access to network
+  - --share=network
+  # Access to X11
+  - --socket=x11
+  # Access to wayland
+  - --socket=wayland
+  # Audio Access
+  - --socket=pulseaudio
+  # Access to all files
+  - --filesystem=host
+  # Access to OpenGL
+  - --device=dri
+  # Use same mouse cursors as host
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --talk-name=org.freedesktop.secrets
+
+rename-desktop-file: asset-manager-studio.desktop
+rename-icon: asset-manager-studio
+modules:
+  - name: ams-desktop
+    buildsystem: simple
+    build-commands:
+      # Extract .deb & cleanup
+      - ar -x asset-manager-studio*.deb
+      - rm -f asset-manager-studio*.deb
```

Comment:
```suggestion
```

Reply:
Why wouldn't we want to delete this file, we don't want to leave it after it is d/l'd.   Or is this not a factor for some reason I don't understand?

Reply:
yep you don't understand that the build dir doesn't exist after the module is build.